src/transformers/models/codegen/modeling_codegen.py

```diff
@@ -475,9 +475,6 @@ def forward(
         if token_type_ids is not None:
             token_type_ids = token_type_ids.view(-1, input_shape[-1])
 
-        if position_ids is not None:
-            position_ids = position_ids.view(-1, input_shape[-1]).long()
-
```

Comment:
hey! Not sure why this was removed as well? Casting to long is probably required / might be breaking if removed!

Reply:
The docstring mentions that `position_ids` should be a tensor of type `torch.LongTensor`, so there should not be a need for the cast.